### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by andynology** [Ankita-Manmoandas/tictactoegame#4]
**UI**
- Fix color contrast on blue "click to play" button. The entire button should be clickable as well, rather than just the text within it.
- Bit more padding on the text inputs. 
- Horizontally center the "Reset" button.

**Code**
- Organise your project structure into a "scripts" and "styles" and "images" folder
- Remove commented code which is no longer needed
- Make sure your code is "indented". Indenting implies scope, which is important.
- Remove unnecessary lines in if statement.. ie.

```jsx
// Unncessary spaces
if (pattern0=="x" && pattern1=="x" && pattern2== "x")  {

	alert ("player 1 wins") ;

//  form.innerHTML= "";

} else if (pattern0=="O" && pattern1=="O" && pattern2== "O")  {

	console.log ("player 2 wins")

}}

// Tidier spaces & indenting
if (pattern0=="x" && pattern1=="x" && pattern2== "x")  {
	alert ("player 1 wins");
} else if (pattern0=="O" && pattern1=="O" && pattern2== "O")  {
	console.log ("player 2 wins")
}}
```

- How can we refactor the checkForWinX functions into a single function with parameters? and make it more DRY?
- Great prefixes like "handleX". However use a "const" rather than a "let" for these functions as they aren't changing.. they are "constant" ;)
- Remove variables which are no longer needed. ie. let winningPattern1
- Remove classes and id's which aren't needed. ie. winningPattern1?
